### PR TITLE
Add some certbot instructions in wiki

### PR DESCRIPTION
**Pull request by Elshirak**
#### What this PR does / why we need it

PR helps users to generate cert for https with certbot. I saw your note [here](https://github.com/jkroepke/openvpn-auth-oauth2/blob/440936ca35bf5392a94a84eec8f834c47d24d072/docs/HTTPS%20Listener.md?plain=1#L65) and wanted to contribute. 

If you have any comments on the PR please give a feedback

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] The PR title has a summary of the changes
- [x] The PR body has a summary to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: docs/HTTPS Listener.md [hunk 1]
```diff
@@ -62,7 +62,27 @@ Alternatively, can use [acme.sh](https://acme.sh/), which is a pure Unix shell s
...
+            --force-renewal \
+            --dns-cloudflare \
+            --dns-cloudflare-credentials /path/to/your/cloudflare.ini \
+            --agree-tos \
+            --email your@email.com \
+            --key-type rsa \
+            --rsa-key-size 2048 \
+            --domains this.is.example.domain \
+            --preferred-challenges dns-01 \
+            --server "https://api.buypass.com/acme/directory"
```
Comment (jkroepke) [inline docs/HTTPS Listener.md:82]: I would recommend the default here.

```suggestion
            --server "https://acme-v02.api.letsencrypt.org/directory"
```

File: docs/HTTPS Listener.md [hunk 2]
```diff
@@ -62,7 +62,27 @@ Alternatively, can use [acme.sh](https://acme.sh/), which is a pure Unix shell s
 
 openvpn-auth-oauth2 requires a [`SIGHUP` signal](https://en.wikipedia.org/wiki/SIGHUP) to reload the TLS certificate.
 
-If you are using certbot, please drop some instructions to setup it.
+## Signing certificates with certbot via dns-challange
+
+You need to install `certbot` and suitable for you `DNS plugin`. More information on supported by certbot DNS plugins and how to config them you may find [here](https://eff-certbot.readthedocs.io/en/stable/using.html#dns-plugins). It's up to you to define what ACME server to use for verification. By default it's set to letsencrypt, you may change it with `--server` [option](https://eff-certbot.readthedocs.io/en/stable/using.html#changing-the-acme-server). Also, take a look at [RFS 8555](https://datatracker.ietf.org/doc/html/rfc8555) about ACME Protocol.
+
+You can create certs with command below and than copy them to `/etc/openvpn-auth-oauth2/` directory or create a link to files. Do not forget to set right permissions.
+
```
Comment (jkroepke) [inline docs/HTTPS Listener.md:70]:
```suggestion

The command below utilize certbot to create an free ssl certificate for a domain hosted on cloudflare.

```

**Review by jkroepke — APPROVED** [review]
Just some nits, otherwise LGTM.

**Comment by Elshirak** [comment]
@jkroepke hello! Thank you for the feedback, I've fixed your suggestions

**Pull request merged**